specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/storage.json

```diff
@@ -2317,8 +2316,8 @@
     "DateAfterModification": {
       "properties": {
         "daysAfterModificationGreaterThan": {
-          "type": "integer",
-          "minimum": 0,
+          "type": "float",
+          "minimum": 0.0,
           "description": "Integer value indicating the age in days after last modification"
```

Comment:
Description is still calling it an integer

Reply:
In the server, the property is defined as float type for testing purpose. But in the real use case, we expect the input to be an integer-like number. I added a constraint in the update https://github.com/Azure/azure-rest-api-specs/pull/6551/commits/0e6f149f5338b7fe8256adb51584a22aaecdb407, which will only allow number that is multiple of 1.

Reply:
Yeah, but since it's no longer an actual int, the docs shouldn't call it that. Maybe just change "Integer value" to "Value"

Reply:
Changed "Integer value" to "Value" https://github.com/Azure/azure-rest-api-specs/pull/6551/commits/0e6f149f5338b7fe8256adb51584a22aaecdb407